centreon/tests/e2e/features/Api-Token/01-api-token-create.feature

```diff
@@ -0,0 +1,39 @@
+@ignore
+@REQ_MON24865
+Feature: Create and Manage Basic API Token
+
+  As an administrator
+  I want to create a functional basic API token using the "Create new token" button
+  So that I can manage access to resources and services
+
+  Background:
+    Given I am logged in as an administrator
+    And I am on the API tokens page
+
+  Scenario: Create Basic API Token with Required Fields
```

Comment:
Is this a scenario with example?
if yes, it should be "Scenario Outline", and parameters should be delimited